lab_5_scrapper/scrapper.py

```diff
@@ -71,14 +152,16 @@ def get_timeout(self) -> int:
         Returns:
             int: Number of seconds to wait for response
         """
+        return self._timeout
 
     def get_verify_certificate(self) -> bool:
         """
-        Retrieve whether to verify certificate.
+        Retrieve number of seconds to wait for response.
```

Comment:
You should not modify functions description

Reply:
I'm sorry, it was not my intention. Fixed everything